### PR TITLE
Update sample cfg to reflect latest choices

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/tilezen/tilequeue/issues/93

@nvkelso, @zerebubuth, @iandees could you review please?

### CODE REVIEW

**Comment by nvkelso** [comment]
Comment look good, I defer to @zerebubuth for the rest.

File: config.yaml.sample [hunk 1]
```diff
@@ -40,30 +53,79 @@ tiles:
...
+  # whether to print out the internal python queue sizes
+  log-queue-sizes: true
+  # and at what interval
+  log-queue-sizes-interval-seconds: 30
+  query-config: <path/to/vector-datasource/queries.yaml>
+  template-path: <path/to/vector-datasource/queries>
+  # whether to reload jinja query templates on each request. This
+  # should be off in production.
+  reload-templates: false
+  # extensions of formats to generate
```
Comment (nvkelso) [inline config.yaml.sample]: We should also comment that buffered MVT (`mvtb`) is possible.

Comment (nvkelso) [inline config.yaml.sample]: Fixed in 47f8a603d3117b833470c2c255dcf180ab019cc8.

**Comment by iandees** [comment]
LGTM

**Comment by zerebubuth** [comment]
:+1:

**Pull request merged**